### ISSUE TITLE
Database Exports Documentation: Color ID really sometimes 8 bit and sometimes 16 bit large?

### ISSUE THREAD

**Issue by Holzhaus** [Deep-Symmetry/crate-digger#27]
In [track rows](https://djl-analysis.deepsymmetry.org/rekordbox-export-analysis/exports.html#track-rows) in PDB files, the color ID field is just a single byte (c<sub>id</sub> at offset 0x58). The same goes for the color index in [Extended (nxs2) Cue List entries](https://djl-analysis.deepsymmetry.org/rekordbox-export-analysis/anlz.html#extended-cue-list) of analysis files  (c<sub>id</sub> at offset 0x1c).

But for PDB [color rows](https://djl-analysis.deepsymmetry.org/rekordbox-export-analysis/exports.html#color-rows), the documentation states that the field is 2 bytes large (id at offset 0x05). Is there a reason to believe that this field is not just also a single byte, and the second byte belongs to u<sub>3</sub> or is yet another unknown byte field?

**Comment by brunchboy** [comment]
Yes, looking at the context a bit, I agree that the color index is probably just a single byte, and that u₃ should just expand to take over the second byte of what is now the color id. Would you be willing to submit a PR to make those changes?
